### DISCUSSION TITLE
How to remove a Wifi entry?

### DISCUSSION THREAD

**Discussion by eigenphase** [sle118/squeezelite-esp32#205] [Q&A]
I have initially connected my MuseLuxe to Wifi1. Then I switched to Wifi2. After reboot, the Luxe is in Wifi2 and stays there, as expected.

But after some time I realize it's back in Wifi1.

How can I remove the credentials for Wifi1 altogether, such that it always just connects to Wifi2?

**Comment by sle118** [comment]
Could you specify which version you are running?

**Reply by eigenphase** [reply]
It’s Muse.16.2.1254.master-v4.3

**Reply by sle118** [reply]
When connected to wifi 1, you can open the web interface and click on the entry. There you get a button to delete the network.

**Reply by 8OND007** [reply]
I had the same problem and could not find a way to delete old Wifi AP connection settings, causing the ESP32 to switch over between 2 AP's  constantly. In the end I completly reflashed the Muze Luxe to clear all Wifi connection credentials. Didn't know there was a button to delete the network,
